### PR TITLE
Add Species-800

### PR DESCRIPTION
**Pull request by edugp**

### CODE REVIEW

File: datasets/species_800/species_800.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        urls_to_download = {
+            "biobert_ner_datasets": _URL,
+        }
+        downloaded_files = dl_manager.download_and_extract(urls_to_download)
+        dataset_directory = f"{downloaded_files['biobert_ner_datasets']}/{_BIOBERT_NER_DATASET_DIRECTORY}"
```
Comment (lhoestq) [inline datasets/species_800/species_800.py:102]:
```suggestion
        dataset_directory = os.path.join(downloaded_files['biobert_ner_datasets'], _BIOBERT_NER_DATASET_DIRECTORY)
```
use `os.path.join` to make it work on windows as well

**Review by lhoestq — APPROVED** [review]
Thanks for adding this one good job !
LGTM

I did one change about path handling for windows

File: datasets/species_800/species_800.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        urls_to_download = {
+            "biobert_ner_datasets": _URL,
+        }
+        downloaded_files = dl_manager.download_and_extract(urls_to_download)
+        dataset_directory = os.path.join(downloaded_files['biobert_ner_datasets'], _BIOBERT_NER_DATASET_DIRECTORY)
```
Comment (lhoestq) [inline datasets/species_800/species_800.py:102]:
```suggestion
        dataset_directory = os.path.join(downloaded_files["biobert_ner_datasets"], _BIOBERT_NER_DATASET_DIRECTORY)

```

File: datasets/species_800/species_800.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""The SPECIES and ORGANISMS Resources for Fast and Accurate Identification of Taxonomic Names in Text"""
+
+import logging
```
Comment (lhoestq) [inline datasets/species_800/species_800.py:19]:
```suggestion
import logging
import os
```

**Comment by edugp** [comment]
thanks @lhoestq ! I probably need to do the same change in the `SplitGenerator`s (lines 107, 110 and 113). I'll open a new PR for that

**Comment by lhoestq** [comment]
Yes indeed ! Good catch 👍 
Feel free to open a PR and ping me

**Comment by obonyojimmy** [comment]
Hi , theres a issue pulling species_800 dataset , throws  google drive  error 

error: 

```
raise ConnectionError(f"Couldn't reach {url} ({repr(head_error)})")
ConnectionError: Couldn't reach https://drive.google.com/u/0/uc?id=1OletxmPYNkz2ltOr9pyT0b0iBtUWxslh&export=download/ (ReadTimeout(ReadTimeoutError("HTTPSConnectionPool(host='drive.google.com', port=443): Read timed out. (read timeout=10)")))
```
code: 
```
from datasets import load_dataset

dataset = load_dataset("species_800")
```

**Comment by edugp** [comment]
Hi @obonyojimmy! I am running the same commands and they work for me. Did you check your internet connection?

**Pull request merged**